api/v1alpha1/pattern_types.go

```diff
@@ -142,34 +140,10 @@ type MultiSourceConfig struct {
 	ClusterGroupChartGitRevision string `json:"clusterGroupChartGitRevision,omitempty"`
 }
 
-type ApplyChangeType string
-
-const (
-	InstallAutomatic ApplyChangeType = "Automatic"
-	InstallManual    ApplyChangeType = "Manual"
-)
-
 type GitOpsConfig struct {
-	// Channel to deploy openshift-gitops from. Default: gitops-1.8
-	// +operator-sdk:csv:customresourcedefinitions:type=spec
-	OperatorChannel string `json:"operatorChannel,omitempty"`
-	// Source to deploy openshift-gitops from. Default: redhat-operators
-	// +operator-sdk:csv:customresourcedefinitions:type=spec
-	OperatorSource string `json:"operatorSource,omitempty"`
-
 	// Require manual intervention before Argo will sync new content. Default: False
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	ManualSync bool `json:"manualSync,omitempty"`
```

Comment:
Why leave this but remove the whole struct from the spec?

Reply:
Oversight.  It has been removed.